MatrixSDK/MXSession.m

```diff
@@ -428,6 +429,20 @@ - (void)startWithSyncFilterId:(NSString *)syncFilterId onServerSyncDone:(void (^
         _store.syncFilterId = syncFilterId;
```

Comment:
I think we should flush the store when the filter id changes

Reply:
Ftm, this is the role of the app. Some filter changes may be applied at runtime. We just log a warning.